packages/next/src/server/base-server.ts

```diff
@@ -711,6 +720,7 @@ export default abstract class Server<ServerOptions extends Options = Options> {
           'http.method': method,
           'http.target': req.url,
         },
+        reqHeaderTraceContext: { traceParent, traceState },
```

Comment:
I don't think we need the intermediary context object. Also let's use the `traceparent` as is, without the camelcase, to follow the OTEL convention

Reply:
>I don't think we need the intermediary context object

To clarify, do you mean to include the traceparent and tracestate properties in the top level of this TracerSpanOptions object?